bbb/servicebase.py

```diff
@@ -18,6 +19,18 @@
 ListenerServiceEvent = namedtuple("ListenerServiceEvent", ("exchange", "routing_key", "callback", "queue_name"))
 
 
+@contextmanager
+def lock_table(db, table_name):
+
+    try:
+        if "mysql" in db.url.get_backend_name():
+            db.execute(sa.text("LOCK TABLE {} WRITE;".format(table_name)))
```

Comment:
I have a vague memory that you cannot use it for table names... Let me double check this.